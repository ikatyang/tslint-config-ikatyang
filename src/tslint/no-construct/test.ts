// good

const x = "something";

// bad

const x = new String("something");
