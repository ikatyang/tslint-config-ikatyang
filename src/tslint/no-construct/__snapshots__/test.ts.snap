// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

const x = "something";

// bad

const x = new String("something");
          ~~~~~~~~~~               [Forbidden constructor, use a literal or simple function call instead]

`;
